src/platform/Ameba/Logging.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <platform_stdlib.h>
+
+namespace chip {
+namespace Logging {
+namespace Ameba {
+
+enum class AmebaLogLevel
```

Comment:
Ameba::AmebaLogLevel stutters? How about just `LogLevel` and then this reads like: `chip::Logging::Ameba::SetLogLevel(chip::Logging::Ameba::LogLevel::kError)`

Reply:
Done, also changed back `chip::Logging::Ameba` to `chip::Logging::Platform` to be consistent with other platforms